hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -597,6 +597,11 @@ private void writeFilesToArchive(
         Path hardLinkFile = createHardLinkList(truncateLength, hardLinkFiles);
         includeFile(hardLinkFile.toFile(), OmSnapshotManager.OM_HARDLINK_FILE,
             archiveOutputStream);
+        try {
```

Comment:
I'd prefer for this to be a try/finally and wrap the create as well.

Reply:
Actually, this whole clause should be in a finally block, with the try wrapping the create.

Reply:
Done.